README.md

```diff
@@ -20,10 +20,12 @@ npm install udaru
 
 ### Stand-alone module
 ```js
-const udaru = require('udaru')
-...
+const buildUdaru = require('udaru')
+const udaru = buildUdaru(dbPool, config)
```

Comment:
Not a fan of the syntax here, would prefer:
`const udaru = require('udaru')(dbPool, config)`